tests/product.py

```diff
@@ -95,6 +96,19 @@ def test_get_all_products(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(len(json_response), 3)
 
-    # TODO: Delete product
+    def test_delete_product(self):
+        """
+        Ensure we can delete an existing product.
+        """
+        url = "/products/1"
+        self.test_create_product()
```

Comment:
The `test_create_product` method is meant to test the create function. The existing Bangazon code also uses it to seed the test db, which is poor practice for a couple of reasons. 1. It violates the Single Responsibility Principle, and 2. it makes the code less 'DRY' (i.e. repetitive, sometimes referred to as 'WET' code).

Instead, there should be code written specifically for seeding the database with product data before each test. The `setUp` method runs before each test in a given test class, which makes it the ideal location for this sort of functionality.

Remove the `test_create_product` method from your `test_delete_a_product` method. Then, modify your `setUp` method to seed product data into the test db before each test. Ping me on Slack once you've pushed up changes, and then you'll be good to go!

Reply:
Let me know if the changes made satisfy the requirement. Thanks